Adyen/UI/Form/Items/Basic Items/FormLabelItem.swift

```diff
@@ -21,24 +22,35 @@ public class FormLabelItem: FormItem {
         self.text = text
     }
 
+    public init(
+        text: String,
+        identifier: String? = nil,
+        labelStyle: LabelStyle = LabelStyle()
+    ) {
+        self.identifier = identifier
+        self.labelStyle = labelStyle
+        self.text = text
+    }
+
     public var identifier: String?
 
     /// The style of the label.
-    public var style: TextStyle
+    public var style: TextStyle = .init(font: .preferredFont(forTextStyle: .body), color: .red)
 
     /// The text of the label.
     public var text: String
 
+    /// The labelStyle from the adyen theme
+    public var labelStyle: LabelStyle = .init()
+
     public func build(with builder: FormItemViewBuilder) -> AnyFormItemView {
         let label = ADYLabel()
         label.text = text
         label.numberOfLines = 0
         label.accessibilityIdentifier = identifier
-        label.font = style.font
-        label.textColor = style.color
-        label.textAlignment = style.textAlignment
-        label.backgroundColor = style.backgroundColor
```

Comment:
This is the moment when we are missing our styling usage telemetry to know if this was even used